packages/engine_umbrella/apps/engine/test/booking_test.exs

```diff
@@ -0,0 +1,73 @@
+defmodule BookingTest do
+  import TestHelper
+  use ExUnit.Case
+
+  test "it allows booking creation" do
+    MessageGenerator.random_booking()
+    |> Booking.make()
```

Comment:
Here we get the newly crated booking ID back, if an ID was returned, surely the booking was created. This would keep the test as a strictly unit-test. If we want to make sure it actually ended up in state, I would propose: Booking.get(id) instead of      booking_ids = Engine.BookingStore.get_bookings() ->  assert booking_ids |> length() == 1

Reply:
And same for the tests that follow the same pattern.

Reply:
:+1: I didn't realize I could just check on an id returned or I can even assert that I get the correct errors